### ISSUE TITLE
Lesson 2 Notebook File is Corrupted 

### ISSUE THREAD

**Issue by hasnainnaeem** [fastai/course-v3#328]
Lesson 2 file "lesson2-download"  is corrupted. It did not work on Jupyter Notebook on my PC (Ubuntu) and on Google Colab.

**Link to notebook:**
https://github.com/fastai/course-v3/blob/master/nbs/dl1/lesson2-download.ipynb

**Comment by kammahm** [comment]
I am having the same problem.

![image](https://user-images.githubusercontent.com/32719626/60216493-0a8e0b00-981f-11e9-984d-94c3475cef19.png)

**Comment by sriharsha-m** [comment]
Facing the same issue on Jupyter and Google colab

![image](https://user-images.githubusercontent.com/39662892/60229153-f1a74900-9862-11e9-94c6-538f6f176f67.png)

**Comment by icj217** [comment]
It looks like line 643 is missing a comma.
```json
"source": [
    "# If the plot is not showing try to give a start and end learning rate",
    "# learn.lr_find(start_lr=1e-5, end_lr=1e-1)",
    ""
    "learn.recorder.plot()"
   ]
```

**Comment by fread2281** [comment]
link to working old version: https://nbviewer.jupyter.org/github/fastai/course-v3/blob/82aba94153fb1fb2e006d1b9d032859bc92a1144/nbs/dl1/lesson2-download.ipynb

**Comment by sgugger** [comment]
Solved by #331 normally
